projector-server-core/src/main/kotlin/org/jetbrains/projector/server/core/util/HttpWsServer.kt

```diff
@@ -121,6 +124,28 @@ public abstract class HttpWsServer(port: Int) {
 
   private val webSocketServer = object : WebSocketServer(InetSocketAddress(port), listOf(HttpDraft(), Draft_6455())) {
 
+    @Volatile
+    var wasInitialized: Boolean? = null
```

Comment:
```suggestion
    private var wasInitialized: Boolean? = null
```

Seems like it can be private.